apps/explorer/src/pages/transaction-result/programmable-transaction-view/AddressOrObject.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useSearch } from '~/hooks/useSearch';
+import { AddressLink, ObjectLink } from '~/ui/InternalLink';
+import { LoadingSpinner } from '~/ui/LoadingSpinner';
+
+interface Props {
+    id: string;
+}
+
+export function AddressOrObject({ id }: Props) {
+    const { data, isLoading, isError } = useSearch(id);
```

Comment:
This is fairly expensive so def don't want to do this. I don't think this is really required though, see my comment below.

Reply:
Yea, you mentioned they are being object, but running it against the test command `pnpm sdk test:e2e` I am getting all of the values as Address. Let me know if the reality is otherwise, or I can always use `AddressLink`.

Reply:
![address2](https://user-images.githubusercontent.com/127577476/225526580-8a77f8d8-72c9-426e-80a9-b42147424df4.gif)

Reply:
Yeah looks like an issue with the API, will follow up